README.md

```diff
@@ -62,8 +62,206 @@ This will open a new jupyter notebook in your localhost where you can run all th
 
 
 
+* [Tic-Tac-Toe](https://github.com/prateekiiest/Code-Sleep-Python#tic-tac-toe)
+* [Hangman](https://github.com/prateekiiest/Code-Sleep-Python#hangman)
+* [Caesar-Cipher](https://github.com/prateekiiest/Code-Sleep-Python#caesar-cipher)
+* [Translations of Hamlet](https://github.com/prateekiiest/Code-Sleep-Python#translations-of-hamlet)
+* [Classification](https://github.com/prateekiiest/Code-Sleep-Python#classification)
+* [Whisky Classification](https://github.com/prateekiiest/Code-Sleep-Python#whisky-classification)
+
+* [Bird Migration](https://github.com/prateekiiest/Code-Sleep-Python#bird-migration)
+* [Social Network Analysis](https://github.com/prateekiiest/Code-Sleep-Python#social-network-analysis)
+* [Prime](https://github.com/prateekiiest/Code-Sleep-Python#prime-number-finder)
+* [Website status check](https://github.com/prateekiiest/Code-Sleep-Python#website-status-check)
+* [Encryption-Techniques](#encryption-techniques)
+* [Inception Tic-Tac-Toe](https://github.com/prateekiiest/Code-Sleep-Python#inception-tic-tac-toe)
+* [Sprint](https://github.com/prateekiiest/Code-Sleep-Python#sprint)
+* [Floating Text](#floating-text)
+* [Koch Curve](https://github.com/prateekiiest/Code-Sleep-Python#koch-curve)
+* [Superellipse](https://github.com/prateekiiest/Code-Sleep-Python#superellipse)
+* [Cricket Notification](https://github.com/prateekiiest/Code-Sleep-Python#cricket-notification)
```

Comment:
Only include the above project links by bullets.
Each project link needs to be updated though